gon.hcl

```diff
@@ -0,0 +1,11 @@
+source = [".build/arm64-apple-macosx/debug/tart"]
+bundle_id = "com.github.cirruslabs.tart"
+
+apple_id {
+  username = "hello@cirruslabs.org"
+  password = "@env:AC_PASSWORD"
+}
+
+sign {
+  application_identity = "Developer ID Application: Fedor Korotkov"
```

Comment:
In order to get rid of `Fedor Korotkov` I've requested to convert the `hello@cirruslabs.org` account to a company one but it's taking a while...